### ISSUE TITLE
Console emulator on the webpage

### ISSUE THREAD

**Issue by nikhildeshmukh** [amabnl/amadeus-ws-client#65]
I want to show the console integrataed in the webpage 
How can we achieve that ?

**Comment by DerMika** [comment]
I'm not sure I understand your question. What console? What do you want to achieve?

**Comment by nikhildeshmukh** [comment]
Its the terminal emulator integrated in the webpage,
I had seen amadeus , sabre in a 3rd party interface, just wondering if we can do the same with this library.

**Comment by DerMika** [comment]
Well, someone wrote a [small app](https://github.com/amabnl/amadeus-ws-client/issues/26) to do cryptic entries in a mimicked terminal screen, maybe you can look at that. You are welcome to investigate this on your own.

I'm not going to investigate this further because it's beyond the scope of this library.

**Comment by DerMika** [comment]
Sorry, I linked the wrong issue, i've updated it now. You can check out that application here: https://github.com/tmconsulting/amadeus-console-emulator

**Comment by nikhildeshmukh** [comment]
i had checked that but there is no support given on that
i had asked a question no one replies there.
question is the same how to show or include the console in the webpage ?

**Comment by DerMika** [comment]
The only way you can (somewhat) emulate the Selling Platform cryptic page via Web Services is by making an application that sends input from a user via `Command_Cryptic` entries to the Web Services server. 

However, doing that will not let you emulate a functional Selling Platform, because it's not meant for that. In fact Amadeus will probably not accept it if you open a project with them with that purpose. 

There are also a number of limitations - some Cryptic entries are forbidden to be sent via `Command_Cryptic` which will limit the usefulness of such an application.

So in short: yes, for some ad hoc testing it's possible to do this as done with the link above, but do not expect to be able to build your own Selling Platform because 1) Amadeus doesn't want you to do that and 2) you will find that certain key entries are not allowed via Command_Cryptic which will limit the functionality of any such solution.
